ide-intellij.html

```diff
@@ -0,0 +1,222 @@
+
+<pre><code>
+Error: JavaFX runtime components are missing, and are required to run this application
+</code></pre>
+
+    This error is shown since the Java <span class="JDK">11</span> launcher checks if the main class extends 
+    <kbd>javafx.application.Application</kbd>. If that is the case, it is required to 
+    have the <kbd>javafx.graphics</kbd> module on the module-path.
+
+</div>
+    
+<p>
+    <strong>4. Add VM options</strong>
+</p>
+<p>
+    To solve the issue, click on <kbd>Run -> Edit Configurations...</kbd> and add these VM options:
+</p>
+
+<pre><code>
+--module-path /path/to/javafx-sdk-<span class="JFX">11</span>/lib --add-modules=javafx.controls,javafx.fxml
+</code></pre>
+    
+<p>    
+    Note that the default project created by IntelliJ uses FXML, so <kbd>javafx.fxml</kbd> 
+    is required along with <kbd>javafx.controls</kbd>. If your project uses other modules, 
+    you will need to add them as well.
+    
+    <a href="images/ide/intellij/ide/idea06.png" target="_blank"><img src="images/ide/intellij/ide/idea06.png" alt="VM options"/></a>
+    
+    Click apply and close the dialog.
+</p>
+
+<p>
+    <strong>5. Run the project</strong>
+</p>
+<p>
+    Click <kbd>Run -> Run...</kbd> to run the project, now it should work fine.
+</p>
+
+<a name="Maven"></a><h3>Maven</h3>
+
+<p>
+    Follow these steps to create a JavaFX project and use the Maven tools to build it and run it.
+</p>
+
+<p>
+    <strong>1. Create a Maven project</strong>
+</p>
+<p>
+    If you select the <kbd>org.codehous.mojo.archetypes:javafx</kbd> archetype, the latest version is from 2015, 
+    so it will rely on the old <kbd>jfxrt.jar</kbd>. But you can modify later on the pom based on this <a href="pom.xml">one</a>.
+    <a href="images/ide/intellij/maven/idea01.png" target="_blank"><img src="images/ide/intellij/maven/idea01.png" alt="Create a Maven project"/></a>
+   
+    Provide the groupId, like <kbd>org.openjfx</kbd>, the artifactId, like <kbd>hellofx</kbd>. 
+    Then provide a name to the project, like <kbd>HelloFX</kbd> and a location for the project.
+    
+    When the project opens, after you synchronize the project, the JavaFX classes are not recognized.
+    
+    <a href="images/ide/intellij/maven/idea02.png" target="_blank"><img src="images/ide/intellij/maven/idea02.png" alt="Missing JavaFX classes"/></a>
+</p>
+
+<p>
+    <strong>2. Modify the pom</strong>
+</p>
+<p>
+    Replace the existing plugins based on this <a href="pom.xml">pom</a> file, and set the 
+    <kbd>mainClass</kbd> accordingly to <kbd>org.openjfx.MainApp</kbd>.
+</p>
+<p>
+    Add the required dependencies for <kbd>javafx.controls</kbd> and <kbd>javafx.fxml</kbd>. 
+    <a href="images/ide/intellij/maven/idea03.png" target="_blank"><img src="images/ide/intellij/maven/idea03.png" alt="Update the pom"/></a>
+   
+    You can download the resulting pom file from <a href="images/ide/intellij/maven/pom.xml">here</a>.
```

Comment:
Use `pom.xml` from samples repository. This will navigate readers to the samples repository which they can then explore.

Reply:
I'll update all the links with the new samples repo